tools/webdriver/webdriver/error.py

```diff
@@ -12,8 +12,8 @@ class WebDriverException(Exception):
     # However, http_status need not match, and both are set as instance
     # variables, shadowing the class variables. TODO: Match on both http_status
     # and status_code and let these be class variables only.
-    http_status = None  # type: ClassVar[int]
-    status_code = None  # type: ClassVar[str]
+    http_status: ClassVar[int]
+    status_code: ClassVar[str]
```

Comment:
This isn't going to work. I tried the same change but something tries to access `WebDriverException.status_code` and that no longer works.

Reply:
yikes. well… that seems even worse, because it implies the type is actually wrong.